plenum/server/replica.py

```diff
@@ -502,11 +502,12 @@ def h(self, n):
 
     @property
     def last_ordered_3pc(self) -> tuple:
-        return self.consensus_provider.get_3pc()
+        return self._consensus_data.view_no, self._consensus_data.pp_seq_no
 
     @last_ordered_3pc.setter
     def last_ordered_3pc(self, key3PC):
-        self.consensus_provider.set_3pc(key3PC)
+        self._consensus_data.view_no = key3PC[0]
```

Comment:
I believe we need to distinguish current viewNo and last ordered 3PC's viewNo, since (because of the presence of audit ledger), we may order 3PC batches from the previous view.

Reply:
ok